src/mesh/Mesh.hpp

```diff
@@ -338,6 +359,18 @@ class Mesh : public PropertyContainer, private boost::noncopyable
    */
   int _globalNumberOfVertices = -1;
 
+  /**
+   * @brief each rank stores list of connected ranks. 
+   * Later in M2N package, this will be used to create initial communication channels. 
+   */
+  std::vector<int> _initialCommunicationMap;
```

Comment:
Same as above. I guess this is not needed.

Reply:
I think we need this one, because we have to pass it to m2n package to create initial channels! But we agreed to rename it to initialConnectionMap!